test/system/autoload_test.rb

```diff
@@ -45,4 +55,14 @@ class AutoloadTest < ApplicationSystemTestCase
       assert_text "Namespace: Hello, from Turbo page"
     end
   end
+
+  def click_button(*)
+    begin
+      assert_css "[data-stimulus-autoloading]"
```

Comment:
This assertion is intended to wait for the autoloading to _start_.

Reply:
It might be best to wrap this in a using_wait_time block of 0-ish to cut down on time spent waiting.